### ISSUE TITLE
Enable jest in CI

### ISSUE THREAD

**Issue by ianb** [mozilla/firefox-voice#992]
I had to disable jest testing (`npm run jest`) in CI (by removing it from `npm test`), because CI couldn't handle the module rewrites. These work for me locally (Node v13.8.0), but CI is acting differently. I _don't_ want any module rewriting when actually using the extension (where the browser has native module support), and especially I don't want it for the old-style modules (including jsx files), it's only necessary for Node-based testing.

**Comment by IkkyOdufade** [comment]
@ianb,Hi,  I'm an intern from Outreachy, I commented on issue #1134 , hoping to work on it. Do I have to fix this issue first?

**Comment by ianb** [comment]
@IkkyOdufade No, this issue is separate, people can still use and test with jest locally so it's good to have instructions on it.

**Comment by IkkyOdufade** [comment]
Hi @ianb. Thank you for the new issue. I’ll start working on it right away.

**Comment by IkkyOdufade** [comment]
Hi @ianb I’m sorry I’ve not created a pr yet. Im working on my laptop. It would be normal again by tomorrow or next. Thank you for understanding.

**Comment by IkkyOdufade** [comment]
Hi, @ianb. I've been trying to work on this and I added `npm run jest` to config.yml. This was the result, the tests passed. I know this is not solving the issue :(, so can I get more help?
![image](https://user-images.githubusercontent.com/47696080/78082086-13150280-73aa-11ea-9c4c-3d3ca9a4699e.png)
